cloudslang-runtime/src/main/java/io/cloudslang/lang/runtime/steps/ActionExecutionData.java

```diff
@@ -55,6 +55,7 @@
     public void doAction(@Param(ScoreLangConstants.RUN_ENV) RunEnvironment runEnv,
                          @Param(ExecutionParametersConsts.NON_SERIALIZABLE_EXECUTION_DATA) Map<String, Object> nonSerializableExecutionData,
                          @Param(ScoreLangConstants.ACTION_TYPE) ActionType actionType,
+                         @Param(ScoreLangConstants.ACTION_DEPENDENCIES) List<String> actionDependencies,
```

Comment:
best practice: for an ExecutionStep parameter the name (`actionDependencies`) should be the same as the `@Param` value (dependencies), however `@Param` counts

I suggest either `actionDependencies` -> `dependencies` or change `ScoreLangConstants.ACTION_DEPENDENCIES` -> `actionDependencies`